docs/_documentations/mdt-che-odo-support.md

```diff
@@ -31,7 +31,7 @@ The extension needs additional roles for accessing OpenShift resources. Use the
 In order to create or import Java compoent, you need to import Java image stream to your OpenShift or OKD cluster so that odo can build component image.
 
 ### Adding the roles and importing the Java image stream
-1. Log in to your OpenShift or Origin Community Distribution (OKD) cluster.
+1. Log in to your OpenShift or Origin Community Distribution (OKD) cluster, and ensure the Codewind workspace has been created.
```

Comment:
- Remove comma after "cluster".
- Use present tense. Instead of "workspace has been created" say "workspace is created."